test/components/NavBar/__snapshots__/OidcLoginComponent.test.jsx.snap

```diff
@@ -1,108 +0,0 @@
-// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
```

Comment:
As we are using material UI more extensively now, these snapshots don't seem to be as useful in testing as I expected them to be. The way material UI writes CSS (composing a series of simple classes) means very small changes to tests results in extreme, indecipherable changes to the snapshot. Both Material UI and react testing library also recommend not doing snapshot testing, so I'm removing this snapshot and just using react testing library to look at the elements we're interested in directly.